src/pfe/file-watcher/server/src/projects/projectUtil.ts

```diff
@@ -967,6 +986,25 @@ export async function isContainerActive(projectID: string, handler: any): Promis
     }
 }
 
+/**
+ * @function
+ * @description Set the ODO app name and save it to the project info file.
+ *
+ * @param projectInfo <Required | ProjectInfo> - The metadata information for a project.
+ *
+ * @returns Promsie<void>
+ */
+export async function setOdoAppName(projectInfo: ProjectInfo): Promise<void> {
+    const projectID = projectInfo.projectID;
+
+    const projectHandler = await projectExtensions.getProjectHandler(projectInfo);
+    const appName = await projectHandler.getAppName(projectID);
```

Comment:
before calling getAppName can we also check if the func is present; because setOdoAppName can be called by any project.. so just want to safeguard...